web/src/tippyjs.js

```diff
@@ -468,4 +468,9 @@ export function initialize() {
         delay: LONG_HOVER_DELAY,
         appendTo: () => document.body,
     });
+
+    delegate("body", {
+        target: ".revoke-invite, resend-invite",
+        appendTo: () => document.querySelector(".organization-box"),
+    });
```

Comment:
This had to be added because the default behaviour for tippy tooltips in Zulip is to appendTo parent. However, that causes the tooltip to get cut when used within the settings/invitations modal's table.

`.organization-box` is the className of the content div of the settings modal.

Reply:
<details>
<summary>Screenshot after the change</summary>
<img src="https://user-images.githubusercontent.com/55971005/229035478-756b9082-349e-45db-aa14-b936e9d3ee87.png">
</details>

Reply:
Had opened [this thread on CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/.E2.9C.94.20tippy.20tooltip.20with.20tables) to discuss the same. Resolved now.